include/giskard_core/yaml_parser.hpp

```diff
@@ -1596,6 +1596,49 @@ namespace YAML {
     }
   };
 
+  //
+  // parsing of alias reference specs
+  //
+
+  inline bool is_alias_reference(const Node& node)
+  {
+    if(!node.IsScalar() || is_const_double(node))
```

Comment:
After some further thinking, the current implementation is correct. The condition uses ```Scalar``` in the sense of ```yaml-cpp```. That means: The node is neither a list nor a map. As references are just strings that is the very first condition. However, double constants, i.e. ```2.5``` are also scalars in the yaml sense. Hence, we have to avoid interpreting them as ```AliasReference```.

To put it differently, a node is shall be parsed as an alias reference, if it is a yaml-scalar and cannot be parsed as a const-double-spec.

Does it now make sense?